backend/kernelCI_app/constants/localization.py

```diff
@@ -13,3 +13,35 @@ class ClientStrings:
     TREE_BOOTS_NOT_FOUND = "No boots available for this tree/branch/commit"
     TREE_NOT_FOUND = "No results available for this tree/branch/commit"
     TREE_TESTS_NOT_FOUND = "No tests available for this tree/branch/commit"
+    TREE_COMMITS_HISTORY_NOT_FOUND = "History of tree commits not found"
+    TREE_STATUS_HISTORY_NOT_FOUND = "Test status history not found"
```

Comment:
Wrong T word :sweat_smile: 
```suggestion
    TEST_STATUS_HISTORY_NOT_FOUND = "Test status history not found"
```